CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,70 @@
+- Fix the checker image tag when running `antctl check cluster` with a released `antctl` binary. ([#6565](https://github.com/antrea-io/antrea/pull/6565), [@tnqn])
+- Use the same MTU as uplink for bridge ports to fix a potential MTU mismatch issue when the traffic mode is changed. ([#6577](https://github.com/antrea-io/antrea/pull/6577), [@antoninbas])
+- Cache TTLs for individual IP addresses in DNS responses to avoid evicting valid IPs before they are expired. ([#6732](https://github.com/antrea-io/antrea/pull/6732), [@hkiiita])
+- Fix an issue with ipset or iptables chain removal during Antrea Node NetworkPolicy updates or deletions. ([#6707](https://github.com/antrea-io/antrea/pull/6707), [@hongliangl])
+- Fix an issue with logging support for L7 NetworkPolicy causing the wrong packet to be logged by Suricata for the default reject rule. From now on, `enableLogging` only controls L4 audit logging and we unconditionally log the packet data for all Suricata alert events. ([#6651](https://github.com/antrea-io/antrea/pull/6651), [@qiyueyao])
+- Fix `NetworkPolicy` related antctl commands including `antctl get networkpolicy` and `antctl get ovsflows`. ([#6487](https://github.com/antrea-io/antrea/pull/6487), [@Dyanngg])
+- Fix the template ID not existing error in IPFIX exporter for `FlowAggregator`. ([#6630](https://github.com/antrea-io/antrea/pull/6630), [@antoninbas])
+- Fix an antrea-agent crash issue when the host interface has been already attached to OVS bridge for `SecondaryNetwork`. ([#6666](https://github.com/antrea-io/antrea/pull/6666), [@xliuxu])
+- Delay the initialization of ARP / NDP responders to fix the `ServiceExternalIP` feature when `SecondaryNetwork` is enabled. ([#6700](https://github.com/antrea-io/antrea/pull/6700), [@xliuxu])
+- Run the `IPPool` webhook handler when `SecondaryNetwork` is enabled. ([#6691](https://github.com/antrea-io/antrea/pull/6691), [@luolanzone])
+- Fix the slice init length issue for `NetworkPolicy` controller. ([#6715](https://github.com/antrea-io/antrea/pull/6715), [@cuishuang])
+- Improve memory copying logic to avoid a potential memory fault issue on Windows. ([#6664](https://github.com/antrea-io/antrea/pull/6664) [#6673](https://github.com/antrea-io/antrea/pull/6673), [@XinShuYang] [@tnqn])
+- Document a workaround for using `EgressSeparateSubnet` feature on OpenShift. ([#6622](https://github.com/antrea-io/antrea/pull/6622), [@luolanzone])
+- Clean up stale resources when `antctl check cluster` failed. ([#6597](https://github.com/antrea-io/antrea/pull/6597), [@luolanzone])
+- Fix hint annotation implementation in `AntreaProxy`. ([#6607](https://github.com/antrea-io/antrea/pull/6607), [@hongliangl])
+- Add initialization of `creationTimestamp` when creating instances of `NodeLatencyStats` to fix a null `creationTimestamp` issue. ([#6574](https://github.com/antrea-io/antrea/pull/6574), [@hkiiita])
```

Comment:
remove "to fix a ..."

Reply:
I refined this sentence to the following:
```
Initialize `creationTimestamp` when creating instances of `NodeLatencyStats` to prevent a null `creationTimestamp` issue
```